handler/src/test/java/io/netty/handler/ssl/ApplicationProtocolNegotiationHandlerTest.java

```diff
@@ -146,6 +177,10 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
         assertArrayEquals(someBytes, channelReadData.get());
         assertTrue(channelReadCompleteCalled.get());
         assertNull(channel.readInbound());
-        channel.finishAndReleaseAll();
+        assertTrue(channel.finishAndReleaseAll());
+    }
+
+    private interface Consumer<T> {
+        void consume(T t);
```

Comment:
nit: If you name this `accept` there will be less to change when we can use `java.util.function.Consumer`.

Reply:
I think that is fine for now